packages/pharos/src/components/link/pharos-link.ts

```diff
@@ -98,22 +98,35 @@ export class PharosLink extends FocusMixin(AnchorElement) {
   }
 
   protected override render(): TemplateResult {
-    return html`<a
-      id="link-element"
-      class="${classMap({
-        [`link--alert`]: this._alert,
-        [`link--hover`]: this._hover,
-      })}"
-      download=${ifDefined(this.download)}
-      href=${ifDefined(this.href)}
-      hreflang=${ifDefined(this.hreflang)}
-      ping=${ifDefined(this.ping)}
-      rel=${ifDefined(this.rel)}
-      target=${ifDefined(this.target)}
-      type=${ifDefined(this.type)}
-      aria-label=${ifDefined(this.label)}
-      @click=${this._handleClick}
-      ><slot></slot>${this.appendContent}</a
-    >`;
+    return this.href
```

Comment:
I think this check is breaking the pagination component since it sets `href=""` which evaluates this to false. This either needs to change to look for the presence of the href attribute `this.hasAttribute('href')` or the pagination component sets its href to something like `#`.

Reply:
Think I prefer the first approach of checking via `hasAttribute`

Reply:
aaah good one.  Made changes, instead of checking for `href` value, now I am checking for attribute existence.